mysql/.gitkeep

```diff
@@ -0,0 +1,4 @@
+  ／l、?             
+（ﾟ､ ｡ ７         
+  l  ~ヽ       
+  じしf_,)ノ
```

Comment:
I think we should create a storage and a database empty directory with this .gitkeep file.